tests/main_test.bal

```diff
@@ -0,0 +1,281 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/lang.runtime;
+import ballerina/log;
+import ballerina/os;
+import ballerina/test;
+
+//Create an endpoint to use Google People API Connector
+GoogleContactsConfiguration googleContactConfig = {oauthClientConfig: {
+        clientId: os:getEnv("CLIENT_ID"),
+        clientSecret: os:getEnv("CLIENT_SECRET"),
+        refreshUrl: REFRESH_URL,
+        refreshToken: os:getEnv("REFRESH_TOKEN")
+    }};
+
+Client googleContactClient = check new (googleContactConfig);
+
+@test:Config {}
+function testListOtherContacts() {
+    log:print("Running List Other Contact Test");
+    string[] readMasks = ["names", "phoneNumbers", "emailAddresses"];
+    var listContacts = googleContactClient->listOtherContacts(readMasks);
+    if (listContacts is stream<Person>) {
+        error? e = listContacts.forEach(isolated function (Person person) {
+            log:print(person.toString());
+        });
+        test:assertTrue(true, msg = "List Other Contacts Failed");
+    } else {
+        test:assertFail(msg = listContacts.message());
+    }
+}
+
+@test:Config {}
+function testCopyOtherContactToMyContact() {
+    log:print("Running copy OtherContact To MyContact Test");
+    string[] copyMask = ["names", "emailAddresses", "phoneNumbers"];
+    string[] readMask = ["names", "emailAddresses", "phoneNumbers"];
+    var copyContacts = googleContactClient->copyOtherContactToMyContact(copyMask, readMask, "otherContacts/c8846080985039646639");
```

Comment:
otherContacts/------
Is it a constant?
Check if this is correct.

Reply:
This thing need to be discussed